lib/libspl/xdr.c

```diff
@@ -29,6 +29,7 @@
  * under license from the Regents of the University of California.
  */
 
+#ifndef WITH_TIRPC
```

Comment:
This #ifdef can go since we will need to reuse our `xdr_control` as I explained below.